common/taskana-common/src/main/resources/sql/db2/taskana-schema-db2.sql

```diff
@@ -68,6 +68,7 @@ CREATE TABLE TASK (
         COMPLETED TIMESTAMP NULL,
         MODIFIED TIMESTAMP NULL,
         PLANNED TIMESTAMP NULL,
+        RECEIVED TIMESTAMP NULL,
```

Comment:
the update script and the schema generate slightly different tables, but if we follow best practices for sql this should not lead to any problems
i can update our sample-data scripts, but this would lead to lower readability and maintainability in my opinion
other solutions could be reordering the table in the update script which would lead to a more complex update script, which would be harder to understand
or i could move the required field to the end of the table, which would disturb the order of columns in my opinion

i would argue, that this does not actually need a fix, as our sample data is only being used by us with our schema and all other database expressions are generated in a way, that ignores actual column order